cbw_api_toolbox/cbw_auth.py

```diff
@@ -8,7 +8,7 @@
 from requests.auth import AuthBase
 
 from cbw_api_toolbox import JSON_CONTENT_TYPE, SIGNATURE_HTTP_HEADER, \
-                            TIMESTAMP_HTTP_HEADER, CONTENT_HASH_HEADER, \
+                            TIMESTAMP_HTTP_HEADER, \
```

Comment:
Remove CONTENT_HASH_HEADER from cbw_api-toolbox if the variable is no more used.